pkg/minikube/bootstrapper/kubeadm/versions.go

```diff
@@ -205,8 +205,17 @@ var versionSpecificOpts = []VersionedExtraOption{
 			Key:       "admission-control",
 			Value:     strings.Join(util.DefaultAdmissionControllers, ","),
 		},
+		LessThanOrEqual:    semver.MustParse("1.10.10"),
 		GreaterThanOrEqual: semver.MustParse("1.9.0-alpha.0"),
 	},
+	{
+		Option: util.ExtraOption{
+			Component: Apiserver,
+			Key:       "enable-admission-plugins",
```

Comment:
Is this necessary? I got a working cluster with just removing the `admission-control` line from `/var/lib/kubeadm.yaml`, so the rest of the default manifest was fine without needing any more overrides.